src/trade-params/TradeParams.js

```diff
@@ -85,6 +85,10 @@ export default class TradeParams extends Component {
         this.onStopTypeChange = ::this.onStopTypeChange;
         this.onStopProfitChange = ::this.onStopProfitChange;
         this.onPurchase = ::this.onPurchase;
+
+        this.state = {
+            dynamicKey: 0,
+        };
```

Comment:
Yeeeah this is bad ;)
We can have the inputs controlled.
By setting defaultValue and just listening on changes we have them uncontrolled.
Setting value should work.